app/digests/api.py

```diff
@@ -24,6 +33,7 @@ class DigestViewSet(viewsets.ModelViewSet):
     pagination_class = DigestPagination
     filter_backends = (DjangoFilterBackend, )
     filter_fields = ('stage',)
+    instance = None
```

Comment:
can this be removed from `self` and just be a local variable? Now its scope is limited to the `with transaction.atomic(): ...`